guava/src/com/google/common/math/DoubleMath.java

```diff
@@ -38,13 +38,16 @@
 import java.math.BigInteger;
 import java.math.RoundingMode;
 import java.util.Iterator;
+import org.checkerframework.checker.index.qual.NonNegative;
+import org.checkerframework.common.value.qual.MinLen;
 
 /**
  * A class for arithmetic on doubles that is not covered by {@link java.lang.Math}.
  *
  * @author Louis Wasserman
  * @since 11.0
  */
+@SuppressWarnings("expression.unparsable.type.invalid")// link to issue: https://github.com/typetools/checker-framework/issues/2030
```

Comment:
This issue is closed, and there was correspondence on it from Mike/Suzanne that you didn't respond to. Why is it still linked here if it's (apparently) no longer an issue?